### ISSUE TITLE
Day of the Gun and 3 character words

### ISSUE THREAD

**Issue by nbdm1284** [michalgm/freedom_archives_admin#26]
The search index does not currently index words 3 characters or smaller, or very common words. Day of the Gun is there, it just won't show up if you search by title.

- Ok, I understand. But perhaps this can be a bit of a larger conversation. I don't want to searches to pull false results based on 3 character words but I do want Day of the Gun to show up when you search for Day of the Gun. Or "Day of the Gun."

**Comment by michalgm** [comment]
Yeah - I'll look into changing the minimum index size to 2 characters. 
It will make searches slower, but I think it shouldn't be too bad given 
the amount of data we have.

**Comment by nbdm1284** [comment]
ok cool, that sounds good.

**Comment by michalgm** [comment]
I tried to change this, but I can no longer log into the cpanel - maybe the password changed? Gimme a call sometime with the proper password and I can fix this.
